scripts/xcode-upload-debug-files.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# print commands before executing them and stop on first error
+set -x -e
+
+echo "warning: uploading debug symbols - set SENTRY_SKIP_DSYM_UPLOAD=true to skip this"
+
+NODE_BINARY=$(command -v node || echo "")
+export NODE_BINARY
+
+IOS_PROJ_PATH=$(pwd)
+
+# Override the default with the global environment
+ENV_PATH="$IOS_PROJ_PATH/.xcode.env"
+if [ -f "$ENV_PATH" ]; then
+    source "$ENV_PATH"
+fi
+
+# Override the global with the local environment
+LOCAL_ENV_PATH="${ENV_PATH}.local"
+if [ -f "$LOCAL_ENV_PATH" ]; then
+    source "$LOCAL_ENV_PATH"
+fi
+
+if [ -z "$NODE_BINARY" ]; then
+  echo "[Warning]: Node path was not found on \`.xcode.env\` and \`.xcode.env.local.\`. " \
+  "You can quickly fix this by going to the path \`${IOS_PROJ_PATH}\`  and running the following script: " \
+  " \`echo export NODE_BINARY=\$(command -v node) > .xcode.env\` " \
+  " Node is required for correctly build your project with Sentry." >&2
+  exit 1
+else
+  echo "Using Node.js from ${NODE_BINARY}"
+fi
+
+# SETUP SENTRY_PROPERTIES
+if [ -z "$SENTRY_PROPERTIES" ]; then
+  # Check if the script is running in the root directory
+  if [ -f "./sentry.properties" ]; then
+    export SENTRY_PROPERTIES=sentry.properties
+  elif [ -f "../../sentry.properties" ]; then
+    export SENTRY_PROPERTIES=../../sentry.properties
```

Comment:
Let's add this to the docs, I assume now we only mention the `ios/sentry.properties` path.

Reply:
I made a new issue to keep track of it https://github.com/getsentry/sentry-cordova/issues/373